COPYING-CC

```diff
@@ -1,4 +1,4 @@
-(c) Copyright 2020 by Coinkite Inc.
+(c) Copyright 2022 by Coinkite Inc.
```

Comment:
not sure If I should have bumped also this..?

Reply:
no I like to leave the year as-is; so it's the first year file existed